components/onboarding_tasklist/onboarding_tasklist.tsx

```diff
@@ -183,11 +191,28 @@ const OnBoardingTaskList = (): JSX.Element | null => {
     const isEnableOnboardingFlow = useSelector((state: GlobalState) => getConfig(state).EnableOnboardingFlow === 'true');
     const [showTaskList, firstTimeOnboarding] = useSelector(getShowTaskListBool);
 
+    // a/b test auto show linked boards
+    const autoShowLinkedBoard = useSelector((state: GlobalState) => shouldShowAutoLinkedBoard(state));
+    const pluginsComponentsList = useSelector((state: GlobalState) => state.plugins.components);
+
     const startTask = (taskName: string) => {
         toggleTaskList();
         handleTaskTrigger(taskName);
     };
 
+    const findRhsPluginId = (pluginId: string) => {
+        const rhsPlugins = pluginsComponentsList.RightHandSidebarComponent;
+
+        if (rhsPlugins.length) {
+            for (const plugin of rhsPlugins) {
+                if (plugin.pluginId === pluginId) {
+                    return plugin.id;
+                }
+            }
+        }
```

Comment:
maybe something like
```suggestion
        return rhsPlugins.find(plugin => plugin.pluginId === pluginId)?.id;
```

Reply:
nice suggestion! thanks!